data/createPages/createContentMenuDataFromPage.js

```diff
@@ -2,10 +2,23 @@ import { isArray } from 'lodash';
 import HtmlDataTypes from '../types/html';
 import { DEFAULT_LANGUAGE } from './constants';
 
-const TYPES_TO_ADD_TO_CONTENT_MENU = [HtmlDataTypes.h2, HtmlDataTypes.h3];
+const TYPES_TO_ADD_TO_CONTENT_MENU = [HtmlDataTypes.h2, HtmlDataTypes.h3, HtmlDataTypes.h6];
 const TYPES_TO_LEVEL_MAP = {
   [HtmlDataTypes.h2]: 2,
   [HtmlDataTypes.h3]: 3,
+  [HtmlDataTypes.h6]: 3,
+};
+
+const httpRESTfulMethods = ['POST', 'GET', 'PUT', 'PATCH', 'DELETE'];
+
+const isItemBeAddedInNav = (name, header) => {
+  if (header === HtmlDataTypes.h6 && name !== '') {
+    const isRestFUL = httpRESTfulMethods.some((method) => name.toUpperCase().startsWith(method));
```

Comment:
Shouldn't this just be `name`? I don't think we want anything starting with `get` to be treated the same as `GET`, it's not really REST associated in the same reliable way